### PR TITLE
Shift planning

### PR DESCRIPTION
**Pull request by RValdre**
Hopefully nothing explodes

### CODE REVIEW

File: front/src/components/ShiftPlanningDialog.js [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+    }
+
+    const handleSnackClose = () => {
+        setSnackOpen(false)
+    }
+
+    const handleSubmit = e => {
+        e.preventDefault()
+        if(formValue.title && formValue.description && formValue.date && formValue.assigned){
+            axios.post("http://localhost:8080/todo/add", {
```
Comment (klillenb) [inline front/src/components/ShiftPlanningDialog.js:61]: fix link and test

File: front/src/components/ShiftPlanningDialog.js [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+            .then(function(response){
+                console.log(response)
+                if(typeof response.data === "object" && response.data !== null){
+                    setSnackOpen(true)
+                    setSnackbarInfo({
+                        text: "Successfully saved!",
+                        severity: "success"
+                    })
+                    setOpen(false)
+                    getData()
```
Comment (klillenb) [inline front/src/components/ShiftPlanningDialog.js:78]: no getData fnc defined

**Review by klillenb — CHANGES_REQUESTED** [review]
ShiftPlanningDialog - fix post link and getData fnc

**Review by klillenb — COMMENTED** [review]
pray...

**Review by klillenb — APPROVED** [review]
pray again

**Review by artjomfrost — APPROVED** [review]

**Pull request merged**